### DISCUSSION TITLE
how do I get 2-digit chapter numbers?

### DISCUSSION THREAD

**Discussion by csells** [audiamus/AaxAudioConverter#211] [General]
I see it's an option to set the number of digits for the series title in the folder structure. How do I set the number of digits for the chapter number for the converted audio files themselves? I need the files to be in dumb alpha order for my mp3 player, so I rename "chapter 1" to "chapter 01" etc by hand after the conversion. Thanks!

**Comment by audiamus** [comment]
Two options here:

- Use the playlist (.m3u) to play the book. The playlist has all tracks in correct order, independent of their name and folder structure.
- If your favourite audio player does not support playlists, you can add a numeric prefix with the correct number of digits to each audio file name. Go to _Basic Settings_, select the  _Chapters_ tab, and in _Use name chapters_ select _Yes, always precede with number_.

**Reply by csells** [reply]
Thanks. I didn't know about the "Yes, always precede with number" options. That did the track.
